### DISCUSSION TITLE
在视频选集中加入总时长的显示

### DISCUSSION THREAD

**Discussion by Bouts2019** [the1812/Bilibili-Evolved#4432] [功能建议 / Ideas]
在一些分p很多的视频中，可以在右侧的视频选集的当前视频的时长的div.duration中，添加一个当前视频以及当前视频之前的视频的累计时长

**Comment by Bouts2019** [comment]
效果类似于
<img width="316" alt="1694652150532" src="https://github.com/the1812/Bilibili-Evolved/assets/48877891/25bc9ddd-241b-4e5f-be98-47952cc15f53">
我自己写了一个简单的油猴脚本，不是很熟悉油猴脚本的开发，供参考
[视频单P总时长](https://greasyfork.org/zh-CN/scripts/475246-%E8%A7%86%E9%A2%91%E5%8D%95p%E6%80%BB%E6%97%B6%E9%95%BF)

**Reply by the1812** [reply]
你这不是已经写好了吗😂
